src/update_handler.c

```diff
@@ -253,10 +253,11 @@ static gboolean casync_extract_image(RaucImage *image, gchar *dest, GError **err
 		seed = g_strdup(seedslot->device);
 	}
 
+extract:
+	/* Set store */
```

Comment:
Good catch, but please be a bit more verbose in your patch discription on why it fixes things. The actual thing that was wrong here is that `casync_extract()` would have been called without having initiallized the `store` variable.